### ISSUE TITLE
1.13.0rc1 requires numpy >=1.16.0 instead of >=1.13.3 stated in setup.py in r1.13 branch???

### ISSUE THREAD

**Issue by henrysky** [tensorflow/tensorflow#25636]
**System information**
- OS Platform and Distribution (e.g., Linux Ubuntu 16.04): Windows 10 x64
- Mobile device (e.g. iPhone 8, Pixel 2, Samsung Galaxy) if the issue happens on mobile device: N/A
- TensorFlow installed from (source or binary): Binary
- TensorFlow version: 1.13.0rc1
- Python version: 3.7.1
- Installed using virtualenv? pip? conda?: pip
- Bazel version (if compiling from source): N/A
- GCC/Compiler version (if compiling from source): N/A
- CUDA/cuDNN version: 10 / 7.4.2
- GPU model and memory: MX150 2GB

**Describe the problem**
Tensorflow 1.13.0rc1 said the numpy requires >=1.13.3 (I had np 1.15.4) in setup.py but it only works with >=1.16.0. Either the setup.py is wrong or there is something wrong while building the wheel??

**Provide the exact sequence of commands / steps that you executed before running into the problem**
```py
import tensorflow
```

**Any other info / logs**
```py
PyBfloat16_Type.tp_base != nullptr
```

**Comment by jvishnuvardhan** [comment]
Sorry for the delay in resolving the issue @henrysky. Could you check whether this is an issue with the latest TF? Thanks!

**Comment by jvishnuvardhan** [comment]
In some cases, I think there are compatibility issues between modules when you install TF using conda. Could you uninstall python and tensorflow and reinstall following the instructions [here](https://github.com/jvishnuvardhan/Installing-TensorFlow-GPU-on-Windows-10-TF1.12). Please let me know how it progresses. Thanks!

**Comment by henrysky** [comment]
I follow the instruction to use official python builds  instead of conda in a VM.

For python 3.6 and 3.7 I have tested, the error still the same when using numpy 1.15, but gone when using numpy 1.16

**Comment by jvishnuvardhan** [comment]
@henrysky Please check required packages [here](https://github.com/tensorflow/tensorflow/blob/master/tensorflow/tools/pip_package/setup.py) for TF1.13.1. In future, double check version of required packages before you install TF through Anaconda. Thanks!

REQUIRED_PACKAGES = [
'absl-py >= 0.7.0',
'astor >= 0.6.0',
'gast >= 0.2.0',
'google_pasta >= 0.1.2',
'keras_applications >= 1.0.6',
'keras_preprocessing >= 1.0.5',
'numpy >= 1.14.5, < 2.0',
'six >= 1.10.0',
'protobuf >= 3.6.1',
'tensorboard >= 1.13.0, < 1.14.0',
'tensorflow_estimator >= 1.13.0rc0, < 1.14.0rc0',
'termcolor >= 1.1.0',
]

**Comment by henrysky** [comment]
This is a clean windows 10 in VM. Using python instead of anaconda python

![image](https://user-images.githubusercontent.com/28623434/53762334-d7052a80-3e95-11e9-8aaa-8af054666c10.png)

**Comment by henrysky** [comment]
@jvishnuvardhan I check with the latest tensorflow 2.0.0a0 same issue is there too

**Comment by combacsa** [comment]
```python
In [1]: import numpy

In [2]: numpy.version
Out[2]: <module 'numpy.version' from '/Users/khbyun/anaconda3/envs/w4/lib/python3.6/site-packages/numpy/version.py'>

In [3]: numpy.version.version
Out[3]: '1.15.1'

In [4]: import tensorflow
---------------------------------------------------------------------------
ModuleNotFoundError                       Traceback (most recent call last)
ModuleNotFoundError: No module named 'numpy.core._multiarray_umath'
---------------------------------------------------------------------------
ImportError                               Traceback (most recent call last)
ImportError: numpy.core.multiarray failed to import

The above exception was the direct cause of the following exception:

SystemError                               Traceback (most recent call last)
~/anaconda3/envs/w4/lib/python3.6/importlib/_bootstrap.py in _find_and_load(name, import_)

SystemError: <class '_frozen_importlib._ModuleLockManager'> returned a result with an error set
---------------------------------------------------------------------------
ImportError                               Traceback (most recent call last)
ImportError: numpy.core._multiarray_umath failed to import
---------------------------------------------------------------------------
ImportError                               Traceback (most recent call last)
ImportError: numpy.core.umath failed to import
2019-03-19 16:46:12.380357: F tensorflow/python/lib/core/bfloat16.cc:675] Check failed: PyBfloat16_Type.tp_base != nullptr
Abort trap: 6
```

According to numpy issue 11871, tensorflow >= 1.13 should depends on numpy >= 1.16 ... 

https://github.com/numpy/numpy/issues/11871

**Comment by combacsa** [comment]
It happens on macOS, too. Not only on windows.

**Comment by facaiy** [comment]
@av8ramit Hi, Amit. Do you know the problem?

**Comment by ctmckee** [comment]
The "ModuleNotFoundError: No module name 'numpy.core._multiarray_umath' occurs when trying to use tensorflowjs_converter

**Comment by Yilong94** [comment]
I can confirm that the error is still occuring. numpy==1.15.4 is not compatible with tensorflow==1.13.1, despite tensorflow v1.13.1 requiring only numpy>=1.13.3

**Comment by mihaimaruseac** [comment]
Going to fix this in 1.13 (with upcoming patch release), 1.14 (with upcoming patch release) and master (for new releases)

**Comment by mihaimaruseac** [comment]
Also notes that there is a workaround where you install numpy before tnesorflow or with the constraint at the same time as tensorflow: `pip install numpy && pip install tensorflow` or `pip install numpy >= 1.16 tensorflow == 1.13`
